test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/AllowListVerificationProviderTests.cs

```diff
@@ -368,26 +343,21 @@ public AllowListVerificationProviderTests(SigningTestFixture fixture)
         }
 
         [CIOnlyFact]
-        public async Task GetTrustResultAsync_VerifyWithoutClientCertificateAllowList_WarnsWhenRequiredAndAllowUntrustedAsync()
+        public async Task GetTrustResultAsync_AuthorSignedPackage_WithoutRepositoryCertificateAllowList_RepositoryAndClientListNotRequired_SuccessAsync()
         {
             // Arrange
             var nupkg = new SimpleTestPackageContext();
 
             using (var dir = TestDirectory.Create())
-            using (var testCertificate = new X509Certificate2(_trustedTestCert.Source.Cert))
+            using (var testCertificate = new X509Certificate2(_trustedAuthorTestCert.Source.Cert))
             {
                 var signedPackagePath = await SignedArchiveTestUtility.AuthorSignPackageAsync(testCertificate, nupkg, dir);
 
-                var trustProviders = new[]
-                {
-                    new AllowListVerificationProvider()
-                };
-
-                var verifier = new PackageSignatureVerifier(trustProviders);
+                var verifier = new PackageSignatureVerifier(_trustProviders);
                 var verifierSettings = GetSettings(
-                    allowUnsigned: false,
-                    allowUntrusted: true,
-                    allowNoClientCertificateList: false,
+                    allowUnsigned: true,
```

Comment:
Why is `allowUnsigned` `true`?

Reply:
This should have no effect for this test, I can change it to false to make sure it fails in the case someone modifies the test making it pass an unsigned package